src/clarity-angular/datagrid/datagrid-action-overflow.spec.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+import {TestContext} from "./helpers.spec";
+import {DatagridActionOverflow} from "./datagrid-action-overflow";
+
+export default function(): void {
+    describe("DatagridActionBar component", function() {
+        let context: TestContext<DatagridActionOverflow, SimpleTest>;
+
+        beforeEach(function() {
+            context = this.create(DatagridActionOverflow, SimpleTest);
+        });
+
+        it("projects content", function() {
+            expect(context.clarityElement.textContent.trim()).toMatch("Hello world");
+        });
+
+        it("opens and closes the menu when the toggle is clicked", function() {
+            let toggle = context.clarityElement.querySelector("clr-icon");
+            expect(context.clarityDirective.open).toBe(false);
+            toggle.click();
+            context.detectChanges();
+            expect(context.clarityDirective.open).toBe(true);
+            toggle.click();
+            context.detectChanges();
+            expect(context.clarityDirective.open).toBe(false);
+        });
+    });
+}
```

Comment:
Please add the following tests (list not exhaustive):
- Register/unregister with the service
- Two-way binding on `[(clrDgActionOverflowOpen)]`
- Close on click outside